src/bp-messages/bp-messages-functions.php

```diff
@@ -254,7 +262,7 @@ function messages_new_message( $args = '' ) {
 	}
 
 	// check if force friendship is enabled and check recipients
-	if ( bp_force_friendship_to_message() && bp_is_active( 'friends' ) ) {
+	if ( bp_force_friendship_to_message() && bp_is_active( 'friends' ) && $is_group_thread <> true) {
```

Comment:
@buddy4kp can you condition like this?
```suggestion
	if ( bp_force_friendship_to_message() && bp_is_active( 'friends' ) && true !== $is_group_thread ) {
```